tsMuxer/hevc.h

```diff
@@ -3,6 +3,8 @@
 
 #include "nalUnits.h"
 
+extern int HDR10_metadata[7];

```

Comment:
This is a global variable that's set by parts of `hevc.cpp` and then read by `blurayHelper.cpp`. Would it be possible to reduce its scope, for example to a member variable of the `BlurayHelper` class?

Reply:
@xavery I've tried for hours to set the variable in a specific class. Then it has to be passed from class to class, I didn't manage to do this and ended up setting the global variable... Maybe someone else will manage to do this.

Reply:
I understand - sadly, I also have no idea on how to get this through in a more civilised way, so I'll just let it go.